app/packs/src/decidim/vocdoni/admin/census.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Toggle sections based on the selected radio button.
+ * @returns {undefined}
+ */
+const toggleSectionsBasedOnRadioButton = function() {
+  const csvRadioButton = document.getElementById("csv_radio_button");
```

Comment:
FYI with the Redesign in v0.28 we're migrating to using data attributes as the javascript selectors, to have a better reparation of concerns. We should try to not add new ones to be aligned with those guidelines and keep consistency between modules and upstream.

Reply:
@andreslucena  Andres, I've made all the changes. It's only in this part of the code that I have doubts. Maybe I misunderstood you. Check it out, please

Reply:
Now it's perfect, at least in this file. Thank you!